Makefile

```diff
@@ -65,6 +66,23 @@ phpstan: vendor $(PHPSTAN)
 validate:
 	composer validate --strict
 
+.PHONY: profile
+profile: 	 ## Runs Blackfire
+profile: vendor tests/benchmark/MutationGenerator/sources tests/benchmark/Tracing/coverage tests/benchmark/Tracing/sources
+	composer dump --classmap-authoritative
+	blackfire run \
+		--samples=5 \
+		--title="MutationGenerator" \
+		--metadata="commit=$(COMMIT_HASH)" \
+		php tests/benchmark/MutationGenerator/profile.php
+	blackfire run \
+		--samples=5 \
+		--title="Tracing" \
+		--metadata="commit=$(COMMIT_HASH)" \
+		php tests/benchmark/Tracing/profile.php
+	composer dump --classmap-authoritative
```

Comment:
why do we need to dump at the end again?

Reply:
completely wrong, I meant to revert it